domains/auth/Auth.API/Tests/Controllers/TestLogoutController.cs

```diff
@@ -63,5 +66,26 @@ public async Task LogoutDeleteCookieReturnSuccess(string? testCookie)
         Assert.Equal(expectedExpiredCookie, logoutController.HttpContext.Response.GetTypedHeaders().SetCookie.Single().ToString());
     }
 
+    [Theory]
+    [InlineData("mitid_user_aborted", "bar?success=0&error_code=E1&error=User%20interrupted")]
+    [InlineData("user_aborted", "bar?success=0&error_code=E1&error=User%20interrupted")]
+    [InlineData("foo", "bar?success=0&error_code=E0&error=Unknown%20error%20from%20Identity%20Provider")]
+    public void OidcProviders_OnOidcFlowFailed(string ErrorDescription, string expectedNextUrl)
```

Comment:
Maybe we can include in the method name why it is supposed to trigger to `OnOidcFlowFailed`. I think we previously had the talk about using this https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html